java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducerConfig.java

```diff
@@ -113,6 +115,10 @@ public static HttpKafkaProducerConfig fromMap(Map<String, Object> map) {
         return new HttpKafkaProducerConfig(hostname, port, topic, sendInterval, messageCount, endpointPrefix);
     }
 
+    public static TracingSystem getTracingSystemFromEnv() {
+        return TracingSystem.forValue(System.getenv().getOrDefault("STRIMZI_TRACING_SYSTEM", ""));
+    }
+
```

Comment:
why didn't you follow the same pattern of the Java HTTP examples? I mean reading the env var in the producer config constructor and filling a class variable and then having the getter for it.
https://github.com/strimzi/client-examples/blob/main/java/http/java-http-producer/src/main/java/HttpProducerConfig.java#L42

The same comment is valid for the consumer side.

Reply:
ConfigRetriever was being used and I could not follow the same pattern as before. I've now removed ConfigRetriever and used the same pattern as before.